src/acl.ts

```diff
@@ -757,6 +757,7 @@ common.util.promisifyAll(Acl, {
   exclude: ['request'],
 });
 
-module.exports = Acl;
-
-module.exports.AclRoleAccessorMethods = AclRoleAccessorMethods;
+export {
```

Comment:
I would shy away from the changes to the exports in the initial PR.  Lets come back around and get these later.